domains/marwan.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "ItsMarwan",
+        "email": "marwan.skins453@gmail.com"
+    },
+    "records": {
+        "CNAME": "itsmarwan.github.io"
+        "TXT": "e2bb275137c3e372f66e5370190e44"
```

if you are going to validate your github site, please refer and read our curated guide -> https://docs.is-a.dev/guides/github-pages/
```suggestion
    "records": {
        "CNAME": "itsmarwan.github.io"
    }
```

---

i do commisions outside of the site not via the site i just add it there if i want to link ppl to the tos, but to be clear i do not have enough knowledge to proccess and handle payments on my own so its on another site not related or linked to this site i do not own, this site is for lets say protfolio and the overview of the stuff i do

---

oh alright i didnt notice, thanks for the heads up, also i updated them both and they should be good to go. check and let me know!